### PR TITLE
Fix obsolete

### PR DESCRIPTION
**Pull request by Villawhatever**
Look ma I actually wrote tests without you asking GOTTEM

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,7 +1,8 @@
+.gitignore
 dsn.txt
 CR.txt
 names.json
 config.json*
 *.gob
 *.exe
-fryatog
\ No newline at end of file
+Fryatog
```
Comment (Fryyyyy) [inline .gitignore]: But why

Comment (Villawhatever) [reply .gitignore]: But why not.

Comment (Fryyyyy) [reply .gitignore]: Turns out gitignore accepts patterns so we can do [fF]ryatog

Comment (Villawhatever) [reply .gitignore]: Oh hey! I... did not know that. Good information.

Comment (Fryyyyy) [reply .gitignore]: Sooooo..can update?

**Pull request merged**